account_reset_chart/models/res_company.py

```diff
@@ -163,4 +152,26 @@ def unlink_from_company(model):
         unlink_from_company('account.tax.code')
         unlink_from_company('account.journal')
         unlink_from_company('account.account')
+
+        try:
+            self.env['account.partial.reconcile'].search(
+                [('company_id', '=', self.id)]
+            ).unlink()
+        except KeyError:
+            pass
+
+        try:
+            self.env['account.full.reconcile'].search(
+                [('company_id', '=', self.id)]
```

Comment:
seems that `company_id` field doesn't exist in `account.full.reconcile` object in V9